singularity/Singularity

```diff
@@ -1,82 +1,97 @@
 Bootstrap:docker
 From:centos:7.4.1708
 
-%environment
-export PATH=/sw/apps/miniconda/bin:$PATH
```

Comment:
This section is moved to the end ... it is how we learnt writing a Singularity recipe!